### PR TITLE
cart item & cart checkout was added

### PR DESCRIPTION
**Pull request by stanislavkulakovskyi**
[https://prnt.sc/eorL6TyrgPMt](screenshot)

### CODE REVIEW

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
+@import '@styles/variables';
+@import '@styles/breakpoints';
+
+.cart_page_title {
+  font-weight: 800;
+  font-size: 32px;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:6]: convert to rem. default is 16px so you can connvert here to 2rem. same down below

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  color: $color-white;
+}
+
+.total_checkout {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 288px;
+  height: 190px;
+  padding: 24px;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:18]: we have variable called main-indent for that. It is exactly 24px

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  border: 1px solid $color-icons;
+
+  @include bp-custom-up (640px) {
+    width: 592px;
+    height: 190px;
+    padding: 24px;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:24]: the same

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    width: 544px;
+  }
+  @include bp-x-large-up {
+    width: 100%;
+  }
+}
+
+.total_price {
+  font-weight: 800;
+  font-size: 32px;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:45]: 2rem

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  line-height: 41px;
+  text-align: center;
+  letter-spacing: -0.01em;
+  color: $color-white;
+}
+
+.number_of_items {
+  font-weight: 600;
+  font-size: 14px;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:54]: convert

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  line-height: 21px;
+  text-align: center;
+
+  color: $color-secondary;
+  order: 1;
+  align-self: stretch;
+  flex-grow: 0;
+  margin-bottom: 16px;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:62]: move to separate variable or use calc($main-indent / 1.5). you use the value of 16px many times

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.line {
+  height: 0;
+  border: 1px solid $color-elements;
+  margin-bottom: 16px;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:68]: same

File: src/components/CartCheckout/CartCheckout.module.scss [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+  text-decoration: none;
+
+  background-color: $color-accent;
+  margin-right: 8px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 48px;
+  color: $color-white;
+
+  transition: background-color 0.2s;
```
Comment (vladyslav-mosor) [inline src/components/CartCheckout/CartCheckout.module.scss:85]: we have extend called transitionDetails. use it here please. it is not so critical to change duration and timing-function

File: src/components/CartItem/CartItem.module.scss [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+@import '@styles/breakpoints';
+
+.cart {
+  display: flex;
+  flex-direction: column;
+  width: 288px;
+  border: 1px solid $color-elements;
+  margin-bottom: 16px;
+  padding: 16px;
+  gap: 24px;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:11]: $main-indent

File: src/components/CartItem/CartItem.module.scss [hunk 10]
```diff
@@ -0,0 +1,147 @@
+@import '@styles/variables';
+@import '@styles/breakpoints';
+
+.cart {
+  display: flex;
+  flex-direction: column;
+  width: 288px;
+  border: 1px solid $color-elements;
+  margin-bottom: 16px;
+  padding: 16px;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:10]: your new variable)

File: src/components/CartItem/CartItem.module.scss [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+  @include bp-custom-up(640px) {
+    flex-direction: row;
+    width: 592px;
+    padding: 24px;
+  }
+
+  @include bp-x-large-up {
+    flex-direction: row;
+    width: 752px;
+  }
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:23]: fd-row should remain from breakpoint 640px but i`m not sure. try to check this out

File: src/components/CartItem/CartItem.module.scss [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+  border: 1px solid $color-elements;
+  margin-bottom: 16px;
+  padding: 16px;
+  gap: 24px;
+  align-items: center;
+
+  @include bp-custom-up(640px) {
+    flex-direction: row;
+    width: 592px;
+    padding: 24px;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:17]: $main-indent

File: src/components/CartItem/CartItem.module.scss [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+    display: flex;
+    align-items: center;
+    width: 100%;
+  }
+}
+
+.delete_button_link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:35]: %flexCenter

File: src/components/CartItem/CartItem.module.scss [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+  padding: 5px;
+  margin-right: 25px;
+  transition: all 0.2s;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:38]: %transitionDetails

File: src/components/CartItem/CartItem.module.scss [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+    transform: scale(1.2);
+  }
+}
+
+.phone_image {
+  width: 66px;
+  height: auto;
+  max-width: 100%;
+  padding: 7px;
+  margin-right: 23px;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:50]: $main-indent

File: src/components/CartItem/CartItem.module.scss [hunk 16]
```diff
@@ -0,0 +1,147 @@
...
+
+  @include bp-custom-up(640px) {
+    width: 200px;
+  }
+}
+
+.quantity_buttons {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:84]: %flexCenter

File: src/components/CartItem/CartItem.module.scss [hunk 17]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+.minus {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 13px;
+  border: 1px solid $color-elements;
+  width: 32px;
+  height: 32px;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:109]: mixin square

File: src/components/CartItem/CartItem.module.scss [hunk 18]
```diff
@@ -0,0 +1,147 @@
...
+
+  transition: background-color 0.2s;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:111]: %transitionDetails

File: src/components/CartItem/CartItem.module.scss [hunk 19]
```diff
@@ -0,0 +1,147 @@
...
+
+  &:active {
+    border: 1px solid $color-secondary;
+  }
+}
+
+.plus {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:125]: %flexCenter

File: src/components/CartItem/CartItem.module.scss [hunk 20]
```diff
@@ -0,0 +1,147 @@
...
+  margin-right: 13px;
+  background-color: $color-elements;
+  width: 32px;
+  height: 32px;
+
+  transition: background-color 0.2s;
```
Comment (vladyslav-mosor) [inline src/components/CartItem/CartItem.module.scss:131]: you already know)

**Review by vladyslav-mosor — CHANGES_REQUESTED** [review]

**Review by rashyd-hasratov — APPROVED** [review]

**Pull request merged**